### PR TITLE
move module to terraform version 0.12

### PR DESCRIPTION
**Pull request by venturel**

### CODE REVIEW

File: sqs_with_iam/output.tf [hunk 1]
```diff
@@ -1,24 +1,25 @@
 output "arn" {
-  value       = ["${aws_sqs_queue.queue.*.arn}"]
+  value       = [aws_sqs_queue.queue.*.arn]
```
Comment (Komorebi-E) [inline sqs_with_iam/output.tf]: Are these brackets needed as the splat will make it a list?

https://www.terraform.io/upgrade-guides/0-12.html#referring-to-list-variables

**Review by Komorebi-E — APPROVED** [review]
As SM, LGTM, maybe change the outputs?

File: sqs_with_iam/output.tf (hunk 1 above)
Comment (iuriaranda) [inline sqs_with_iam/output.tf]: No need for a list notation in these outputs afaik

**Review by iuriaranda — APPROVED** [review]
Just a small change in the outputs. Other than that it looks ok to me

**Pull request merged**